frontend/revive-gallery-app/src/App.js

```diff
@@ -14,6 +15,7 @@ function App() {
           <Route path="/login" element={<LoginForm />} />
```

Comment:
@SaiKiran0407 Update "/" path to render homepage component instead of RegistrationForm component

Reply:
sure